README.RDAP.md

```diff
@@ -4,21 +4,31 @@ Read more about the RDAP specification in the RFC documents on the IETF site: ht
 
 Multiple country attributes are not returned
 --------------------------------------------
-inetnum and inet6num objects can contain multiple country attributes, but RDAP only allows a single value.
+inetnum and inet6num objects can contain multiple country attributes, but RDAP schema only allows a single value.
 
-This implementation returns the first country attribute value, and includes an explanatory notice.
+This implementation returns the first country attribute value, and includes the field as redacted.
 
 Multiple language attributes are not returned
 ---------------------------------------------
-inetnum, inet6num, and organisation objects can have multiple language attributes, but only the first language is returned.
+inetnum, inet6num, and organisation can contain multiple language attributes, but RDAP schema only allows a single
+value.
+
+This implementation returns the first language attribute value, and include the field as redacted.
 
 Multiple organisation phone attributes are returned, but not with preferences
 ----------------------------------------------------------------------------------------
-Preferences are not assigned to multiple phone elements.
+Preferences are not assigned to multiple phone elements. The prioritise of the attributes in RPSL objects are unknown, therefore it
+is not possible to figure out the priority of the phone elements.
+
+According to [nro-rdap-profile](https://bitbucket.org/nroecg/nro-rdap-profile/raw/v1/nro-rdap-profile.txt) preference parameters are MAY.
```

Comment:
Re-word as "According to the RFC, the preference is optional".

(In that case, maybe we shouldn't document that we don't include an *optional* value? I'm fine to remove this section).